CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.19.2
+### Fixed
+- Task `Security Enlightn` will now only tell what composer packages have security issues, but will not block
```

Comment:
Can you share why we don't want this to block anymore?

Reply:
I now get the error that magento has a security issue if a new patch is released or if we have clients with older magento versions.
Like this for example:
magento/product-community-edition (2.4.4)
-----------------------------------------

 * CVE-2024-34102: Arbitrary Code Execution through Improper Restriction of XML External Entity Reference (XXE) vulnerability
   https://helpx.adobe.com/security/products/magento/apsb24-40.html